config/blast.php

```diff
@@ -3,9 +3,54 @@
 return [
     'storybook_server_url' =>
         env('STORYBOOK_SERVER_HOST', env('APP_URL')) . '/storybook_preview',
+    
+    /**
+     * See https://storybook.js.org/docs/react/essentials/controls Set
+     * to true to enable full documentation on the controls tab. 
+     */
+    'storybook_expanded_controls' => true,
 
-    // See https://storybook.js.org/docs/react/configure/theming for available options
-    'storybook_theme' => [],
```

Comment:
Sorry for the all the back and forth on this. I was thinking about this some more and I'm worried that this will introduce a breaking change. If someone is defining themes using the old approach, this update will break their theme.  Can we detect if this is an array in the `manager.js` and if so load it the old way?